<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-CHD9VV8" failures="3" tests="5" name="Test" time="43.203" errors="0" timestamp="2024-11-01T11:50:18 IST">
  <testcase classname="testCases.TC_003_LoginDDT" name="verify_loginDDT" time="11.265">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC_003_LoginDDT.verify_loginDDT(TC_003_LoginDDT.java:54)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC_003_LoginDDT" name="verify_loginDDT" time="4.791">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC_003_LoginDDT.verify_loginDDT(TC_003_LoginDDT.java:63)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC_003_LoginDDT" name="verify_loginDDT" time="4.022">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC_003_LoginDDT.verify_loginDDT(TC_003_LoginDDT.java:63)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC_003_LoginDDT" name="verify_loginDDT" time="12.191"/>
  <testcase classname="testCases.TC_003_LoginDDT" name="verify_loginDDT" time="3.041"/>
</testsuite> <!-- Test -->
